HACKING.md

```diff
@@ -44,6 +44,7 @@ In addition, for testing, the following dependencies are required:
     $ sudo yum install curl expect \
         libvirt libvirt-client libvirt-daemon libvirt-python \
         python python-libguestfs python-lxml libguestfs-xfs \
+	python3 libvirt-python3 \
```

Comment:
We still need libvirt-python in the list for test/common/testvm.py, so for the time being we indeed need both versions.

Reply:
Roger. Yes, we have both versions.